### ISSUE TITLE
tIED: ImportIED Communication section duplicates each ConnectedAP

### ISSUE THREAD

**Issue by danyill** [OpenEnergyTools/scl-lib#45]
Here is an example ICD file:

[XAT_SEL-487E-5_StandardDesign.ICD.zip](https://github.com/OpenEnergyTools/scl-lib/files/12653148/XAT_SEL-487E-5_StandardDesign.ICD.zip)

I opened: [test_comms.scd.zip](https://github.com/OpenEnergyTools/scl-lib/files/12653177/test_comms.scd.zip) in https://danyill.github.io/oscd-import-templates/index.deploy.html which is a prototype which I'm testing using this library (and may still be the cause of bugs...).

When I saved the file I saw only the first ConnectedAP (see #44) but it was there twice.

I think the cause of this is these lines:

https://github.com/OpenEnergyTools/scl-lib/blob/864bb51c14137a04d70199d9df3d0e83a9531b07/tIED/insertIED.ts#L39-L40

Because the insertIED _moves_ rather than _copies_ the IED, I don't think we need to use `importNode` which creates a second object reference.

**Comment by JakobVogelsang** [comment]
@danyill do you use the `importIED` function more than one time. So do you import more than one IED. If so, you have to careful. Initially I have tried to build an `importIEDs` function which turned out to be a nightmare in complexity and I ended up building this one instead. You can try to do something like this

```ts
const actionsBatch1 = importIED(scl,temaplte1);
this.dispatchEvent(newEditEvent(actionsBatch1));
await this.editCount.updateCompete;
const actionsBatch2 = importIED(scl,temaplte2);
this.dispatchEvent(newEditEvent(actionsBatch1)); 
```

the functions that are build in `scl-lib` are valid always against an updated `doc`. Otherwise, it will not work really.

**Comment by JakobVogelsang** [comment]
I can confirm it being incorrect on [scl-editor](https://openenergytools.github.io//scl-editor). Sorry for this. I will push a fix!

**Comment by JakobVogelsang** [comment]
Fixed with version 0.5.10

https://github.com/OpenEnergyTools/scl-lib/commit/384df9b3545f1831ca41b58c4e35bde9813d2538

**Comment by danyill** [comment]
The solution looks good, thanks :+1:
